docs/content/contributing/documentation.md

```diff
@@ -1,8 +1,3 @@
----
-title: Documentation
-type: docs
----
-
```

Comment:
Here you can see most of all the difference in formatting between Hugo and MkDocs